Modules/System/Cryptography Management/src/CryptographyManagementImpl.Codeunit.al

```diff
@@ -610,4 +609,90 @@ codeunit 1279 "Cryptography Management Impl."
             Error(GetLastErrorText());
         Hash := Convert.ToBase64String(ByteArray);
     end;
+
+    procedure EncryptTextWithDESCryptoServiceProvider(VarInput: Text; Password: Text; Salt: Text) VarOutput: Text
+    var
+        ByteArray: DotNet Array;
+        SymmetricAlgorithm: DotNet Cryptography.SymmetricAlgorithm;
+        Encoding: DotNet Encoding;
+    begin
+        if VarInput = '' then
+            exit;
+
+        FuncConstructDESCryptoServiceProvider(SymmetricAlgorithm, Password, Salt);
```

Comment:
Are we missing this function ?

Reply:
The name of the function was incorrect - removed the Func (in upcoming PR)